src/WPF/WPF.Viewer/Samples/Analysis/QueryFeatureCountAndExtent/QueryFeatureCountAndExtent.xaml

```diff
@@ -12,28 +12,30 @@
     </UserControl.Resources>
```

Comment:
```
<UserControl
    x:Class="ArcGIS.WPF.Samples.QueryFeatureCountAndExtent.QueryFeatureCountAndExtent"
    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
    xmlns:esri="http://schemas.esri.com/arcgis/runtime/2013">
    <Grid>
        <esri:MapView x:Name="MyMapView" />
        <Border MaxWidth="300" Style="{StaticResource BorderStyle}">
            <StackPanel>
                <TextBlock
                    Grid.Row="0"
                    FontWeight="SemiBold"
                    Text="Select a state to zoom to matching features:"
                    TextWrapping="Wrap" />
                <ComboBox
                    x:Name="StatesComboBox"
                    Grid.Row="1"
                    Margin="0,5"
                    HorizontalAlignment="Stretch"
                    SelectionChanged="StatesComboBox_SelectionChanged" />
                <Button
                    Grid.Row="2"
                    Margin="0,0,0,5"
                    HorizontalAlignment="Stretch"
                    Click="CountFeaturesButton_Click"
                    Content="Count features in extent" />
                <TextBlock
                    x:Name="Results"
                    Grid.Row="3"
                    IsEnabled="False"
                    TextAlignment="Center" />
            </StackPanel>
        </Border>
    </Grid>
</UserControl>
```

This solves the different `Button` and `ComboBox` width issue.

Reply:
Nice solve!